.pre-commit-config.yaml

```
@@ -20,7 +20,7 @@ repos:
     rev: v0.13.0
     hooks:
       - id: ruff-check
-        args: [ --fix, --show-fixes ]
+        args: [ --show-fixes ]
```

```suggestion
        args: [ --fix, --show-fixes ]
```

Oops, using this locally.